docs/DeveloperGuide.md

```diff
@@ -257,27 +257,55 @@ _{Explain here how the data archiving feature will be implemented}_
 
 
 ### User stories
 
 Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unlikely to have) - `*`
 
-| Priority | As a …​                                    | I want to …​                     | So that I can…​                                                        |
-| -------- | ------------------------------------------ | ------------------------------ | ---------------------------------------------------------------------- |
-| `* * *`  | new user                                   | see usage instructions         | refer to instructions when I forget how to use the App                 |
-| `* * *`  | user                                       | add a new person               |                                                                        |
-| `* * *`  | user                                       | delete a person                | remove entries that I no longer need                                   |
-| `* * *`  | user                                       | find a person by name          | locate details of persons without having to go through the entire list |
-| `* *`    | user                                       | hide private contact details   | minimize chance of someone else seeing them by accident                |
-| `*`      | user with many persons in the address book | sort persons by name           | locate a person easily                                                 |
+| Priority | As a …                | I can …                                                                                                  | So that …                                                                                                                                       |
+|----------|-----------------------|----------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------|
+| `*`      | medical administrator | export a current week’s schedule for nurses so that                                                      | they do not need to contact me for their schedules repeatedly                                                                                   |
+| `* * *`  | medical administrator | quickly add the name, age, address and contact of the patients that require home visits so that          | retrieve them later                                                                                                                             |
+| `* * *`  | medical administrator | quickly add the name, age, address and contact of the nurse/volunteers                                   | retrieve them later for matching                                                                                                                |
```

Comment:
Hmmm maybe can just write "quickly add a patient with respective details with or without tag"

Same as nurse, and maybe scrap the volunteer first? because we are not implementing at all or like put it as a separate user stories

Reply:
Yeah I think we can leave out the volunteers for now, and add them later if we have time!

Reply:
Okay i will make the changes!